components/var_dumper.rst

```diff
@@ -272,26 +272,32 @@ Example::
     {
         use VarDumperTestTrait;
 
-        protected function setUpVarDumper(array $casters, int $flags = null): void
+        protected function setUp()
         {
-            $this->varDumperConfig['casters'] = [
+            $casters = [
                 \DateTimeInterface::class => static function (\DateTimeInterface $date, array $a, Stub $stub): array {
                     $stub->class = 'DateTime';
                     return ['date' => $date->format('d/m/Y')];
                 },
             ];
 
-            $this->varDumperConfig['flags'] = CliDumper::DUMP_LIGHT_ARRAY | CliDumper::DUMP_COMMA_SEPARATOR;
+            $flags = CliDumper::DUMP_LIGHT_ARRAY | CliDumper::DUMP_COMMA_SEPARATOR;
+
+            // if the custom casters and flags apply only to some tests, call to this method
+            // from those tests instead of the setUp() method, which applies to all tests
```

Comment:
@javiereguiluz could you reword this 2 lines comment, its hard to read IMO

Reply:
```suggestion
            // Configure the casters & flags to use for all the tests in this class.
            // If you need custom casters per test cases rather than for the whole class,
            // call this method from each of these test cases instead.
```